lib/screens/tree_list/tree_list_view_model.dart

```diff
@@ -86,7 +86,15 @@ abstract class TreeListViewModelBase with Store, ViewModel {
   }
 
   void connectionChanged(dynamic hasConnection) {
-    hasInterNetConnection = hasConnection;
+    hasInternetConnection = hasConnection;
     fetch();
   }
+
+  void activateConnectionManager() {
+    connectionStatus.initialize();
+    //Listen for connection change
+    connectionStatus.connectionChangeStream
+        .asBroadcastStream()
+        .listen(connectionChanged);
+  }
```

Comment:
Not necessary ( like the "connectionChanged" method) because it will be already initialised in the main_common file